### DISCUSSION TITLE
DataFlex not identified as programming language

### DISCUSSION THREAD

**Discussion by SoftAllan** [community/community#37180] [Repositories]
![image](https://user-images.githubusercontent.com/7941239/197705961-938526c3-4353-442b-aaf0-0c93fd1aa4d1.png)

Example repository: https://github.com/SoftAllan/Filesystem

Data Access Worldwide it the company behind the DataFlex development language https://www.dataaccess.com/products/dataflex-63

**Comment by ossanna16** [comment]
Hi there @SoftAllan and welcome to our community! Thank you for your post. Can you please elaborate a bit further on your idea?

**Reply by seanbamforth** [reply]
Hi 

Let me jump in on this too. We both of us use a venerable, but largely unknown language called dataflex. There are a few repos on Github that use dataflex. 

What's the best way to train github copilot on this language. It kind of works sometimes, 
How can we make it useful for dataflex? 

I'm happy to talk about this in detail. Not sure that this is a usefult thing for Guthub, but would be super useful for me. 
Though it can't be made public, I have a couple of million lines of code I could throw at a language model.
